content/components/overlay.mdx

```diff
@@ -0,0 +1,383 @@
+        style="background: none"
+    />
+</ImageContainer>
+<Caption>An anchored overlay attached to its source.</Caption>
+
+Keep anchored overlays out of the viewport edges. They should be at least 16px away from the viewport boundaries. Users shouldn't be blocked from scrolling the page while an anchored overlay is open, nor should a scrolling event in the page close the overlay.
+
+For more complex overlays, consider enabling mobile-specific bottom sheet or full-screen variations.
+
+Primer React uses [`anchoredPosition`](https://primer.style/components/anchoredPosition) to calculate the position of an anchored or centered overlay:
+
+<ImageContainer>
+  <img
+    width="508"
+    src="https://user-images.githubusercontent.com/293280/126408752-c40d6f55-3790-45e7-9ea6-529c702154da.png"
+    alt=" "
+    style="background: none"
+  />
+</ImageContainer>
+<Caption>
+
+A visualization of available [`anchoredPosition`]((https://primer.style/components/anchoredPosition)) options. Source elements (purple squares) and their overlays (white squares) with  `side` (`outside-top`, `outside-right`, `outside-bottom`, `outside-left`) and `align` (`start`, `center`, `end`) settings.
+
+</Caption>
+
+### Centered overlays as dialogs
+
+Dialogs appear centered in the page, with a visible backdrop that dims the rest of the window for focus.
+
+All dialogs should have a title and a close button. Use an overlay header to compose these required elements.
+
+Dialogs are modal. Dialogs can be dismissed by clicking on the close button, or by interacting with another button in the overlay. To avoid losing information and missing important messages, clicking outside of the dialog will not close it. Prevent page scrolling when a dialog is visible.
+
+Make sure larger dialogs remain mobile-friendly. Even large dialogs need to be responsive. A dialog's width and height will be readjusted depending on the screen size and should never exceed the available space. Use responsive solutions to adjust the UI so they behave well on smaller screens.
+
+Dialogs never hit the viewport edges. Use a safe 32px margin to keep dialogs visually distinct from the rest of the page.
+
+Simple and small dialogs can remain as-is on mobile. As more elements are added to it, mobile-specific style variations such as Bottom sheet and Full-screen should be considered.
+
+<ImageContainer>
+  <img
+    width="838"
+    src="https://user-images.githubusercontent.com/293280/126410311-da5b64f2-049b-4eca-a5d8-2281b49f9d43.png"
+    alt=" "
+    style="background: none"
+  />
+</ImageContainer>
+<Caption>
+
+A visualization of `inside-center` positioning, with underneath source elements and their available `align` (`start`, `center`, `end`) settings. 
+
+</Caption>
+
+### Bottom sheets (specific to small screens)
+
+A bottom sheet is a mobile-friendly and mobile-specific placement option. Overlays that appear anchored or centered in large screens may adapt to appear as a bottom sheet on small screens.
```

Comment:
If we don't ever want a bottom sheet to be used on a large viewport, we should explicitly say that.